packages/webapp/src/components/Badge/index.tsx

```diff
@@ -0,0 +1,100 @@
+/*
+ *  Copyright 2024 LiteFarm.org
+ *  This file is part of LiteFarm.
+ *
+ *  LiteFarm is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  LiteFarm is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ *  GNU General Public License for more details, see <https://www.gnu.org/licenses/>.
+ */
+
+import React, { useState } from 'react';
+import { Tooltip, ClickAwayListener, IconButton } from '@mui/material';
+import InfoOutlinedIcon from '@mui/icons-material/InfoOutlined';
+import styles from './styles.module.scss';
+
+// Define the Props
+interface BadgeProps {
+  content: React.ReactNode;
+  title: string;
+  showIcon?: boolean;
+}
+
+const Badge: React.FC<BadgeProps> = ({ content, title = 'Beta', showIcon = true }) => {
+  const [open, setOpen] = useState<boolean>(false);
+  const [tooltipHover, setTooltipHover] = useState<boolean>(false);
+
+  const handleTooltipOpen = () => {
+    if (!tooltipHover) setOpen(true);
+  };
+  const handleTooltipClose = () => {
+    if (!tooltipHover) setOpen(false);
+  };
+  const handleTooltipContentHover = (hovering: boolean) => {
+    setTooltipHover(hovering);
+    setOpen(hovering);
+  };
+
+  const tooltipContent = (
+    <div
+      onClick={() => handleTooltipContentHover(true)}
+      onMouseEnter={() => handleTooltipContentHover(true)}
+      onMouseLeave={() => handleTooltipContentHover(false)}
+      className={styles.tooltip}
+    >
+      {content}
+    </div>
+  );
+
+  const popperProps = {
+    modifiers: [
+      {
+        name: 'offset',
+        options: {
+          offset: [0, -10],
+        },
+      },
+    ],
+  };
```

Comment:
This offset is causing the popover to overlap with the tooltip contents, which causes the hover styling not to work properly for a small portion of the tooltip when it's open. To get rid of the spacing between the popover and the tooltip, I'd suggest replacing this with the margin change alternative documented here in MUI docs https://v5.mui.com/material-ui/react-tooltip/#distance-from-anchor (where it says "Alternatively, you can use the slotProps prop to customize the margin of the popper."). According to the designs, there should still be a subtle margin between the badge and the popover, of 2px.